networking/openshift-sdn/about-openshift-sdn.adoc

```diff
@@ -0,0 +1,27 @@
+[id="about-openshift-sdn"]
+= About OpenShift SDN
+include::modules/common-attributes.adoc[]
+:context: about-openshift-sdn
+
+toc::[]
+
+{product-title} uses a software-defined networking (SDN) approach to provide a
+unified cluster network that enables communication between pods across the
+{product-title} cluster. This pod network is established and maintained by the
+OpenShift SDN, which configures an overlay network using Open vSwitch (OVS).
+
+OpenShift SDN provides three SDN modes for configuring the pod network:
+
+* The _network policy_ mode allows project administrators to configure their own
+isolation policies using `NetworkPolicy` objects. Network policy is the default
+mode in {product-title} {product-version}.
+
+* The _multitenant mode_ provides project-level isolation for pods and services.
+Pods from different projects cannot send packets to or receive packets from pods
+and services of a different project. You can disable isolation for a project,
+allowing it to send network traffic to all Pods and services in the entire
+cluster and receive network traffic from those Pods and services.
+
+* The _subnet_ mode provides a "flat" pod network where every pod can
+communicate with every other pod and service. The _network policy_ mode provides
```

Comment:
You've already introduced "network policy" before, so shouldn't italicize it here now.